packages/get-balance-stream/test-app/truffle-config.js

```diff
@@ -0,0 +1,7 @@
+const path = require("path");
+
+module.exports = {
+  // See <http://truffleframework.com/docs/advanced/configuration>
+  // to customize your Truffle configuration!
+  contracts_build_directory: path.join(__dirname, "client/src/contracts")
```

Comment:
Man, I'd really like to use nano-react-app instead so we don't have to do this funky thing.

Reply:
I think that should be part of another PR to refactor everything.